### PR TITLE
Sessions

### PR DESCRIPTION
**Pull request by ymnj**
Closed for new branch with user testing

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <%= button_tag( :class => "btn waves-effect waves-light" ) do %>
+        Login <i class='material-icons right'>send</i>
+      <% end %>
+    </div>
+  </div>
+    <% end %> <!-- End form -->
+</div>
+
+
+
```
Comment (zibs) [inline app/views/sessions/new.html.erb]: Do we need all this whitespace or am I just snooty? XD

File: config/routes.rb [hunk 2]
```diff
@@ -14,6 +14,15 @@
   # Example resource route (maps HTTP verbs to controller actions automatically):
   #   resources :products
 
+  # Users
+  resources :users, except: [:index, :new]
```
Comment (zibs) [inline config/routes.rb]: might as well just make this

``` ruby
resources :users
```

**Comment by tkbeili** [comment]
Specs are empty! please add some specs.

File: lib/templates/erb/scaffold/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
```
Comment (tkbeili) [inline lib/templates/erb/scaffold/_form.html.erb:1]: This is not a valid ERB you have double `%` signs. Also, check the rest of the code in this file.

File: lib/templates/erb/scaffold/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  <div class="form-inputs">
+  <%- attributes.each do |attribute| -%>
+    <%%= f.<%= attribute.reference? ? :association : :input %> :<%= attribute.name %> %>
+  <%- end -%>
+  </div>
+
+  <div class="form-actions">
+    <%%= f.button :submit %>
+  </div>
+<%% end %>
```
Comment (tkbeili) [inline lib/templates/erb/scaffold/_form.html.erb]: Generally speaking avoid using scaffolding in this project. It usually adds many files that we don't need.

File: lib/templates/erb/scaffold/_form.html.erb (hunk 3 above)
Comment (tkbeili) [inline lib/templates/erb/scaffold/_form.html.erb:1]: Sorry I didn't notice that this is from SimpleForm, please keep it :)

File: app/views/sessions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+<div class="container">
+
+  <h4 class="center-align">User Login</h4>
+
+  <div class="row">
+    <%= form_tag sessions_path, {:class => 'col s12 l6 offset-l3'} do %>
```
Comment (tkbeili) [inline app/views/sessions/new.html.erb:6]: no need for the Hash rocket in here

File: app/views/users/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+<div class="container">
+
+  <div>
+    <%= @user.errors.full_messages.join(", ") %>
+  </div>
+
+  <h4 class="center-align">User Signup</h4>
+
+  <div class="row">
+  <%= form_for @user, html: {class: "col s12"} do |f| %>
```
Comment (tkbeili) [inline app/views/users/new.html.erb:11]: You've set up simple form so please use it here

**Pull request closed without merging**